handlers/main.yml

```diff
@@ -4,9 +4,5 @@
   action: ansible.builtin.setup
 
 - name: Reload and restart rclone-mount services
-  ansible.builtin.shell: |
-    systemctl daemon-reload
-    for service in /etc/systemd/system/rclone-mount@*.service; do
-      systemctl enable $(basename $service)
-      systemctl restart $(basename $service)
-    done
\ No newline at end of file
+  ansible.builtin.systemd:
```

Comment:
You know that this does not enable and restart the services? It's not the same behavior.